src/main/java/at/hannibal2/skyhanni/config/features/CommandsConfig.java

```diff
@@ -56,11 +58,24 @@ public static class TabCompleteConfig {
         public boolean gfsSack = true;
     }
 
+    @ConfigOption(name = "Fandom Wiki for §e/wiki", desc = "")
+    @Accordion
     @Expose
-    @ConfigOption(name = "Fandom Wiki", desc = "Use Fandom wiki (§ehypixel-skyblock.fandom.com§7) instead of the Hypixel wiki (§ewiki.hypixel.net§7).")
-    @ConfigEditorBoolean
-    @FeatureToggle
-    public boolean useFandomWiki = false;
+    public FandomWikiCommmandConfig fandomWiki = new FandomWikiCommmandConfig();
+
+    public static class FandomWikiCommmandConfig {
+
+        @Expose
+        @ConfigOption(name = "Use Fandom", desc = "Use Fandom wiki (§ehypixel-skyblock.fandom.com§7) instead of the Hypixel wiki (§ewiki.hypixel.net§7).")
+        @ConfigEditorBoolean
+        @FeatureToggle
+        public boolean useFandomWiki = false;
```

Comment:
Please add config migration for this. also maybe replace the name to something better, like "replace chat" and use this as the name of this toggle as well

Reply:
done